guides/doc-Upgrading_Project/master.adoc

```diff
@@ -53,4 +53,9 @@ ifdef::foreman-el,katello,satellite,orcharhino[]
 // Upgrading the External database
 include::common/modules/proc_upgrading-the-external-database.adoc[leveloffset=+2]
 endif::[]
+
+ifdef::satellite[]
```

Comment:
This should be rendered for all but `foreman-deb` please.
Only `satellite-clone` part is Satellite-specific.